### ISSUE TITLE
deprecation: multi-channel independent pitch-bend no longer working

### ISSUE THREAD

**Issue by wolftune** [mtytel/helm#210]
This feature came from issue https://github.com/mtytel/helm/issues/64 originally and worked great until recent updates made it stop working.

I don't know if the MPE stuff will use or subsume this feature (since MPE includes note-specific / channel-specific pitch bend) as I'm not sure on the implementation. But bringing back the previous functioning would be really good anyway, independently or in conjunction with MPE support.

**Comment by mtytel** [comment]
Right, I don't think I'm going to add MPE to helm but should definitely fix this..
